travel_platform/travel_app/templates/travel_app/tour_detail.html

```diff
@@ -0,0 +1,28 @@
+{% extends "travel_app/base.html" %}
+{% load static from staticfiles %}
+{{ detail }}
+{% block content %}
+
+  {% for attraction in detail %}
+  <h1>{{ attraction.tour_title }}</h1>
+    <h3>{{ attraction.title }}</h3>
+
+    {% for image in attraction.images %}
+      <div>
+        <img src="{{ image }}">
```

Comment:
It's a good manner to add `alt` parameter.
So, if we may extract any description from `image` please add it.

Thanks!